src/__tests__/union-class-names-test.js

```diff
@@ -1,8 +1,6 @@
-/*global jest, describe, it, expect*/
+/* global jest, describe, it, expect*/
 
-jest.dontMock('../lib/utils/union-class-names');
```

Comment:
jest.dontMock ??
but why did you removed it only for '../lib/utils/union-class-names'